aim/sdk/objects/image.py

```diff
@@ -101,29 +106,52 @@ def json(self):
             'height': self.height,
         }
 
-    def _from_pil_image(self, pil_image: PILImage.Image):
+    def _from_pil_image(self, pil_image: PILImage.Image, compress=False):
```

Comment:
since this and the rest of `_from_*` methods are called from `__init__` where we explicitly pass compress, no need for default value.